kvbc/include/categorization/kv_blockchain.h

```diff
@@ -170,19 +169,24 @@ class KeyValueBlockchain {
  public:
   struct KeyValueBlockchain_tester {
     void instantiateCategories(KeyValueBlockchain& kvbc) { kvbc.instantiateCategories(); }
-    const std::map<std::string, std::variant<detail::ImmutableKeyValueCategory, BlockMerkleCategory, KVHashCategory>>&
+    const std::map<std::string,
```

Comment:
Nit: Can use `auto` to save some typing.

Reply:
I like it :)